### PR TITLE
License: add missing notices related to eclipselink

### PR DESCRIPTION
**Pull request by adutra**
# Description

Build is failing when property `eclipseLink` is true:

```text
./gradlew clean build -PeclipseLink=true
> Task :polaris-service:generateLicenseReport FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':polaris-service:generateLicenseReport'.
> License information for the following artifacts is missing in the root LICENSE file: 
```

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Documentation update
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] `./gradlew clean build -PeclipseLink=true` now passes

# Checklist:

Please delete options that are not relevant.

- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] If adding new functionality, I have discussed my implementation with the community using the linked GitHub issue

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -2110,6 +2334,255 @@ do so.  If you do not wish to do so, delete this exception statement
 from your version.
 
 
+---
+com.sun.xml.bind:jaxb-xjc
```
Comment (snazy) [inline LICENSE:2338]: Just move this like below line 1352, if the license is exactly the same.

File: LICENSE [hunk 2]
```diff
@@ -2892,34 +3365,411 @@ permitted.
...
+
+Content may contain encryption software. The country in which you are currently
+may have restrictions on the import, possession, and use, and/or re-export to
+another country, of encryption software. BEFORE using any encryption software,
+please check the country's laws, regulations and policies concerning the import,
+possession, or use, and re-export of encryption software, to see if this is
+permitted.
+
+---
+jakarta.xml.bind:jakarta.xml.bind-api
```
Comment (snazy) [inline LICENSE:3745]: Same here - the licenses for Jakarta look the same

File: LICENSE (hunk 1 above)
Comment (adutra) [reply LICENSE:2338]: To me it doesn't look the same, on line 2371 there is a mention `Copyright (c) 2004 Kohsuke Kawaguchi` that is not present in `jaxb-core` license.

File: LICENSE (hunk 2 above)
Comment (adutra) [reply LICENSE:3745]: From what I see that are divergences:

```
jakarta.annotation:jakarta.annotation-api
jakarta.el:jakarta.el-api
jakarta.servlet:jakarta.servlet-api
jakarta.ws.rs:jakarta.ws.rs-api

  => EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0

jakarta.xml.bind:jakarta.xml.bind-api

  => BSD-3-Clause (+Oracle preamble?)

jakarta.persistence:jakarta.persistence-api (new)

  => EPL-2.0 OR BSD-3-Clause
```

File: LICENSE (hunk 1 above)
Comment (snazy) [reply LICENSE:2338]: ok

File: LICENSE (hunk 2 above)
Comment (snazy) [reply LICENSE:3745]: ok

**Review by snazy — APPROVED** [review]

**Pull request merged**